### ISSUE TITLE
Could not install on Jetson Nano aarch64 [SOLVED!]

### ISSUE THREAD

**Issue by Bogula** [tamuhey/tokenizations#21]
Built wheel for CPython 3.6m to /home/ai/lib/tokenizations/python/target/wheels/pytokenizations-0.6.0-cp36-cp36m-manylinux1_aarch64.whl

could not install WHEEL

ERROR: pytokenizations-0.6.0-cp36-cp36m-manylinux1_aarch64.whl is not a supported wheel on this platform.

Hardware: Jetson Nano , 18.04, Jetpack 4.4
spaCy version: 2.3.2
Platform: Linux-4.9.140-tegra-aarch64-with-Ubuntu-18.04-bionic
Python version: 3.6.9

Thanks
 Werner

**Comment by tamuhey** [comment]
@Bogula Hi, thanks for reporting issue!

What command did you exec?

**Comment by Bogula** [comment]
Dear tamuhey,

thanks for asking back. I finally found a solution:

Rebuild tokenizations module from source.
$ git clone https://github.com/tamuhey/tokenizations
$ cd python
$ pip install maturin
$ maturin build

pip will fail to install on platform Jetson Nano with the resulting whl - filename:

"pytokenizations-0.6.0-cp36-cp36m-manylinux1_aarch64.whl"

As a result the .so file will not land in the tokenizations folder

If we change just the name of the wheel file to pytokenizations-0.6.0-cp36-none-any.whl
and install with

sudo pip3 install pytokenizations-0.6.0-cp36-none-any.whl

it will work. 

Maybe you have to force-reinstall.

Finally you'll find the .so file in the tokenizations folder and it will work.

Thank you
 Werner

**Comment by tamuhey** [comment]
@Bogula Thanks for sharing the solution!

Will it fail to simply `pip install pytokenizations`?

**Comment by tamuhey** [comment]
I've reproduced this issue with `docker.io/aarch64/python:latest` container.
Re-open this issue and will look for simple workaround.

**Comment by Bogula** [comment]
pip said something like: "cannot install wheel file for this architecture. "

So it installed the __init__.py etc files, but does not copy the .so file to the tokenization folder.

after renaming the wheel-file to: pytokenizations-0.6.0-cp36-none-any.whl

sudo pip3 install pytokenizations-0.6.0-cp36-none-any.whl 

did the install. 

I also added  --force-reinstall.

I don't know renaming  alone did the trick, or the --force-reinstall or both,
but after applying both I found all files in tokenizations folder and the module worked.

Cheers
 Werner
